lib/platform/bitbucket/index.spec.ts

```diff
@@ -1,4 +1,7 @@
+// TODO fix mocks
+/* eslint-disable jest/expect-expect */
 import URL from 'url';
+import 'jest-extended';
```

Comment:
Do we really need this here? Should be added to a [`global.d.ts`](https://github.com/jest-community/jest-extended#typescript)

or import it here instead of `jest.config.js`
https://github.com/renovatebot/renovate/blob/c1f9c416617ead3c2fb31c04a3190960aa9ca5cd/test/globals.ts#L1

Reply:
It's what they suggested in their README, but I couldn't get it to work. I might just remove it, as I only use it in a couple of places.